src/test/java/hw6/steps/ThenSteps.java

```diff
@@ -0,0 +1,90 @@
+package hw6.steps;
+
+import hw6.entities.DropdownTable;
+import hw6.entities.LogrowTable;
+import hw6.entities.UserTable;
+import io.cucumber.java.DataTableType;
+import io.cucumber.java.en.Then;
+
+import java.util.List;
+import java.util.Map;
+
+import static org.testng.Assert.assertEquals;
+import static org.testng.Assert.assertTrue;
+
+public class ThenSteps extends BaseSteps {
+    @Then("{string} page should be opened")
+    public void browserTitleShouldBe(String string) {
+        assertEquals(epamUserIndexPage.getBrowserTitle(), string);
+    }
+
+    @Then("User name should be {string}")
+    public void usernameShouldBe(String string) {
+        assertEquals(epamUserIndexPage.getUserNameText(), string);
+    }
+
+    @DataTableType
+    public LogrowTable logrowTable(Map<String, String> entry){
+        return new LogrowTable(entry.get("value"), entry.get("condition"));
+    }
+
+    @Then("LogRow should contain following values:")
+    public void logrowShouldContain(List<LogrowTable> logrowTables){
+        for(int i = 0; i < logrowTables.size(); i++){
+            assertTrue(differentElementsPage.logRowsElementIsDisplayed(logrowTables.get(i).getLogrowValue(),
+                    logrowTables.get(i).getLogrowCondition()));
+        }
```

Comment:
May be ti will be better set `logrowTable` as parameter for the `differentElementsPage.logRowsElementIsDisplayed()` method?

Reply:
done